sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/AvroIO.java

```diff
@@ -678,6 +687,22 @@ public PDone apply(PCollection<T> input) {
                     filenamePrefix, filenameSuffix, shardTemplate, AvroCoder.of(type, schema))));
       }
 
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        builder.add("schema", type);
+        if (filenamePrefix != null) {
+          builder.add("fileNamePrefix", filenamePrefix);
+        }
+
+        if (filenameSuffix != null && !filenameSuffix.isEmpty()) {
+          builder.add("fileNameSuffix", filenameSuffix);
```

Comment:
This is incomplete -- look at every function that returns a `new Bound`, Missing shard name template, etc.

I wonder if, in `FileBasedSink`, we could just assemble the entire filename (template) and use that.